configure.ac

```diff
@@ -201,25 +201,7 @@ AC_CHECK_HEADERS([sys/time.h])
 
 if test "x$platform" = xposix; then
 	dnl check availability of clock_gettime()
-	if test "x$backend" = xdarwin; then
-		dnl need to verify that OS X target is 10.12 or later for clock_gettime()
-		AC_MSG_CHECKING([whether OS X target version is 10.12 or later])
-		AC_COMPILE_IFELSE([AC_LANG_PROGRAM([
-				#include <AvailabilityMacros.h>
-				#if MAC_OS_X_VERSION_MIN_REQUIRED < MAC_OS_X_VERSION_10_12
-				# error "Target OS X version is too old"
-				#endif
-			], [])],
-			[AC_MSG_RESULT([yes])
-			 osx_10_12_or_later=yes],
-			[AC_MSG_RESULT([no])
-			 osx_10_12_or_later=])
-		if test "x$osx_10_12_or_later" = xyes; then
-			AC_CHECK_FUNCS([clock_gettime], [have_clock_gettime=yes], [have_clock_gettime=])
-		else
-			AC_MSG_NOTICE([clock_gettime() is not available on target OS X version])
-		fi
-	else
+	if test "x$backend" != xdarwin; then
```

Comment:
I appreciate the minimum invasiveness, but these two tests could be &&'d for readability.

Reply:
Which two tests?

Reply:
```
if test "x$platform" = xposix; then
	dnl check availability of clock_gettime()
	if test "x$backend" != xdarwin; then
```

Reply:
But doesn't the `dnl` statement need to run for the first test but not the second?

Reply:
Oh damn is that a comment?!

Reply:
I think it might get something printed, but if you are not checking for the availability it shouldn't display either.

Reply:
OK, I've taken another stab.

Reply:
stackoverflow to the rescue: dnl stands for “Discard to Next Line”  (so pure comment)

Reply:
Yeah, found similar. What a weird comment indicator.